fiatconnect-api.md

```diff
@@ -1890,51 +1894,88 @@ the UNIX timestamp that the request was sent. The second will be of the form `s=
 MUST be computed from the shared webhook private key and a dot-separated string consisting of the UNIX timestamp joined with the request body.
 This signature verification design is based off of [Persona's webhook documentation](https://docs.withpersona.com/docs/best-practices).
 
-# 6. AML Considerations
+# 6. Amounts
+
+In the interest of preventing rounding errors, [floating point errors](https://floating-point-gui.de/), or similar errors, numerical amounts 
+MUST be communicated as strings for every request parameter and every response field throughout the FiatConnect API. 
+
+Clients and servers alike SHOULD NOT cast fiat or crypto amounts to floating point numbers or integers, but should instead
+consider libraries such as [bignumber.js](https://github.com/MikeMcl/bignumber.js/) for handling large numbers and decimals 
+to an arbitrary number of decimal places.
+
+## 6.1 Units
+
+Except where otherwise noted, amounts of cryptocurrencies MUST be given in the units of that currency, e.g. `1.5` Celo means
+one and a half Celo (not `1.5e-18` Celo, which would be the case if amounts were instead given in 
+[wei](https://www.investopedia.com/terms/w/wei.asp#:~:text=What%20Is%20a%20Wei%3F,one%20quintillionth%20of%20an%20ether.)).
+
+Amounts of fiat currencies MUST be given in the units of that currency, e.g. `5.00` USD means five USD.
+
+## 6.2 Decimal Places
+
+Given that amounts are communicated in divisible units-- such as US Dollars, which are divisible into `100` cents, or Celo, which 
+is divisible into `10^18` wei--  it is expected that the client or server will sometimes need to communicate non-integer amounts. 
+For all amounts, [decimal notation](https://en.wikipedia.org/wiki/Decimal#Decimal_notation) MUST be used-- for example, 
+One half should be given as `0.5`.
+
+## 6.3 Precision
+
+FiatConnect-compliant API's MUST observe 18 decimal places of precision for all cryptocurrencies and 2 decimal places of
```

Comment:
Ah, you call out precision specifically here. Maybe a bit nit-picky, but ERC20 tokens don't necessarily have 18 decimals of precision; it's less common, but I've run into it plenty. No reason not support those tokens in the future IMO.

Reply:
updated

Reply:
didn't realize that about erc20